src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -77,16 +76,29 @@ function VisEditor($scope, $route, timefilter, AppState, $location, kbnUrl, $tim
   let stateMonitor;
   const $appStatus = this.appStatus = {};
 
+  // Retrieve the resolved SavedVis instance.
   const savedVis = $route.current.locals.savedVis;
 
+  // Instance of vis.js.
```

Comment:
I think that in order for this to be useful the comment should point to the actual file, not just `vis.js`